lib/redis_memo/memoize_query/cached_select.rb

```diff
@@ -336,17 +337,19 @@ def self.extract_bind_params_recurse(node)
       end
 
       bind_params
-
-    when Arel::Nodes::NotEqual
-      # We don't cache based on NOT queries (where.not) because it is unbound
-      # but we memoize queries with NOT and other bound queries, so we return the original bind_params
-      return bind_params
     else
       # Not yet supported
       return
     end
   end
 
+  # Whether the expression of the node has the compare operators
+  def self.is_comparison_expr(expr)
+    return false if !expr.is_a?(Arel::Nodes::SqlLiteral)
+    compare_operators = ['>', '>=', '<', '<=', '!=']
+    compare_operators.any?{ |comparator| expr.to_s.include?(comparator) }
```

Comment:
```suggestion
   %w[< >= < <= !=].include?(expr.to_s)
```
at some point, we should add some rubocop lint rules to this repo :)

Reply:
I think this is a reverse relationship, but I used the %w